coordinateutils/src/get_interp_val.c

```diff
@@ -80,9 +80,10 @@ void get_ring_info(map_info *minfo, long iring, long *startpix,
   Allocate and initialize map_info structure.
 
   nside : map dimension
+  shift_phi : if 1, center rings at phi = 0 instead of phi = pi
   populate : if 1, populate all ring parameters
 */
-map_info * init_map_info(size_t nside, int populate) {
+map_info * init_map_info(size_t nside, int nest, int shift_phi, int populate) {
```

Comment:
Same comment about int vs. bool (C99 has bool).

Reply:
Since this is the only C code in the library that would require C99, I'm going to just leave things as they are here.  We can revisit if we integrate more C code later on.